it/controllers/AddressLookupConfirmPageISpec.scala

```diff
@@ -22,20 +22,24 @@ class AddressLookupConfirmPageISpec extends IntegrationSpecBase {
       val res = await(fResponse)
       val doc = getDocFromResponse(fResponse)
 
-      doc.getElementsByClass("back-link").first().text() shouldBe "Back"
-      doc.getElementsByTag("title").first().text() shouldBe "Confirm the address"
-
-      doc.getElementById("pageHeading").text() shouldBe "Review and confirm"
-      doc.getElementById("changeLink").text() shouldBe "Edit this address"
-      doc.getElementById("continue").text() shouldBe "Confirm and continue"
-      doc.getElementById("line1").text() shouldBe "1 High Street"
-      doc.getElementById("line2").text() shouldBe "Line 2"
-      doc.getElementById("line3").text() shouldBe "Line 3"
-      doc.getElementById("line4").text() shouldBe "Telford"
-      doc.getElementById("postCode").text() shouldBe "AB11 1AB"
-      doc.getElementById("country").text() shouldBe "France"
-      doc.getElementsByTag("h2").select(":containsOwn(Your selected address)").size() shouldBe 0
-      doc.getElementsByTag("p").select(":containsOwn(This is how your address will look. Please double-check it and, if accurate, click on the Confirm button.)").size() shouldBe 0
+      doc.select("a[class=back-link]") should have(
+        text("Back")
+      )
+      doc.title shouldBe CONFIRM_PAGE_TITLE
+      doc.h1.text() shouldBe CONFIRM_PAGE_HEADING
+      doc.submitButton.text() shouldBe "Confirm and continue"
+      doc.link("changeLink") should have(
+        text(CONFIRM_PAGE_EDIT_LINK_TEXT)
+      )
+      doc.address should have (
+        addressLine("line1", "1 High Street"),
+        addressLine("line2", "Telford"),
+        addressLine("postCode", "AB11 1AB"),
+        addressLine("country", "France")
+      )
+      doc.paras should not contain "This is how your address will look. Please double-check it and, if accurate, click on the Confirm button."
```

Comment:
This will need refactoring to reflect the new format, e.g.:

`doc.paras should not have(elementWithValue("This is how your address will look. Please double-check it and, if accurate, click on the Confirm button.")`

At the moment it's passing because it's not finding a match, but it's not looking at the right thing so not a true test.